### ISSUE TITLE
Easy to make explorer.exe stuck or crash when connect/disconnect from remote desktop

### ISSUE THREAD

**Issue by liyafe1997** [torchgm/RoundedTB#198]
![1](https://user-images.githubusercontent.com/18359157/161048033-e333875a-b696-498a-9581-a33a3ce62e0e.jpg)

When connect/disconnect(re-login in computer screen) from remote desktop, sometimes the taskbar will lost icon(shows a blank slot), got stucked, even whole explorer.exe stucked or crash. I think the problem easy to happen when changing the resolution and disconnecting monitors. (Like I have 2 monitors connected to my computer, but when I use remote desktop to connect it, the two monitors will become one and also resolution changed.

Here is my rtb.log when the above screenshot happened.

[rtb.log](https://github.com/torchgm/RoundedTB/files/8389022/rtb.log)
)

**Comment by PureLo** [comment]
Yes I have the same problem, when I'm using two monitors and after using Microsoft's remote desktop interface, RoundedTB on my secondary screen doesn't work
![屏幕截图(1)](https://user-images.githubusercontent.com/65882628/210290774-9b4f2c0f-acdc-4620-91a8-92646845ac05.png)
